contracts/handler/extensible/TokenCallbacks.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.7.0 <0.9.0;
+
+import {ERC1155TokenReceiver} from "../../interfaces/ERC1155TokenReceiver.sol";
+import {ERC721TokenReceiver} from "../../interfaces/ERC721TokenReceiver.sol";
+
+import {ExtensibleBase} from "./ExtensibleBase.sol";
+
+/**
+ * @title TokenCallbacks - ERC1155 and ERC721 token callbacks for Safes
+ * @author mfw78 <mfw78@rndlabs.xyz>
+ * @notice Refactored from https://github.com/safe-global/safe-contracts/blob/3c3fc80f7f9aef1d39aaae2b53db5f4490051b0d/contracts/handler/TokenCallbackHandler.sol
+ */
+abstract contract TokenCallbacks is ExtensibleBase, ERC1155TokenReceiver, ERC721TokenReceiver {
```

Comment:
this contract doesn't seem to be used anywhere, and it is marked as abstract, why is it needed?

Reply:
This is used with inheritance in `ExtensibleFallbackHandler`.